### DISCUSSION TITLE
How to use Mujoco to create a Stewart Platform

### DISCUSSION THREAD

**Discussion by 13302931942** [google-deepmind/mujoco#2117] [Asking for Help]
### Intro

Hi!

I am a Mujoco enthusiast, I use MuJoCo for my research on Stewart Platform，but I don't know how to create a parallel robot。I hope that all experts can provide a Stewart Platfrom Mujoco model. If there is a simulation process, it would be even better
Thank you

### My setup

Mujoco3.2.3
Windows 10
python3.9

### Minimal model and/or code that explain my question

If you encountered the issue in a complex model, please simplify it as much as possible (while still reproducing the issue).

Model:
<details>
<summary>minimal XML</summary>

```XML

<mujoco>
  <worldbody>
    <light pos="0 0 1"/>
    <geom type="sphere" size="1" rgba="1 0 0 1"/>
  </worldbody>
</mujoco>

```
</details>

Code:
```python
import mujoco
import mediapy as media
model = mujoco.MjModel.from_xml_string(xml)
data = mujoco.MjData(model)

with mujoco.Renderer(model) as renderer:
  mujoco.mj_forward(model, data)
  renderer.update_scene(data)

  media.show_image(renderer.render())
```

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by quagla** [comment]
Hi, you can use constraints to create a parallel robot. See for example this parallel gripper https://github.com/google-deepmind/mujoco_menagerie/tree/main/robotiq_2f85
